elements/bulbs-poll/fields/selected-answer.test.js

```diff
@@ -14,17 +14,17 @@ describe('<bulbs-poll> SelectedAnswerField', function () {
     let { selectAnswer } = actions;
 
     it('selects first answer', function () {
-      let nextState = selectAnswer.invoke(null, answer1);
+      let nextState = selectAnswer(null, answer1);
```

Comment:
I really like this. It seems much more intuitive to simply call a function than have to prefix it with `invoke`